### DISCUSSION TITLE
이모티콘 보내기 어렵네요...(js)

### DISCUSSION THREAD

**Discussion by ghost** [storycraft/node-kakao#439] [Q&A]
![image](https://user-images.githubusercontent.com/86477017/123430524-3fe2b480-d603-11eb-80df-7c25fc98dd82.png)
이렇게만 작동되고 이모티콘은 안보내집니다
제코드에 무슨문제가 있는건가요

**Comment by ghost** [comment]
console.log로 result를 확인해보세요.

**Reply by ghost** [reply]
어떻게 하는건가요

**Reply by ghost** [reply]
해결했습니다 감사합니다!

**Reply by tadashi74** [reply]
> 해결했습니다 감사합니다!

어떻게 해결하셨는지 여쭤봐도 될까요?
